tests/test-local-dev-recipes.bicep

```diff
@@ -0,0 +1,202 @@
+import radius as radius
+
+param magpieimage string
+
+import kubernetes as kubernetes {
+  namespace: 'daprrp-rs-secretstore-recipe'
+  kubeConfig: ''
+}
+
+resource env 'Applications.Core/environments@2023-10-01-preview' = {
+  name: 'local-dev-recipe-env'
+  location: 'global'
+  properties: {
+    compute: {
+      kind: 'kubernetes'
+      resourceId: 'self'
+      namespace: 'local-dev-recipe-env'
+    }
+    recipes: {
+      'Applications.Datastores/mongoDatabases':{
+        mongoazure: {
+          templateKind: 'bicep'
+          plainHTTP: true
+          templatePath: 'reciperegistry:5000/recipes/local-dev/mongodatabases:latest'
```

Comment:
Could we make `reciperegistry:5000` a parameter with a default value so it could be overridden if needed?

Reply:
And could we make `latest` a parameter as well?

Reply:
done